web/src/beta/lib/core/StoryPanel/Block/builtin/Text/index.tsx

```diff
@@ -32,10 +32,11 @@ const TextBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
     },
     [block?.property?.id, block?.property?.items, handlePropertyValueUpdate],
   );
+  const title = useMemo(() => block?.property?.items?.[1]?.title, [block?.property?.items]);
```

Comment:
Since we already are passing block.property.items to the BlockWrapper, can you move this in to the wrapper? That way we don't need to remember to write this useMemo inside EACH builtin block's component 🙏🏻